src/app.jsx

```diff
@@ -45,16 +86,27 @@ export default class App extends React.Component {
   async componentDidMount() {
     const investList = await getInvestModelNames();
     const recentJobs = await InvestJob.getJobStore();
-    // TODO: also load and set investSettings from a cached state, instead
-    // of always re-setting to these hardcoded values on first launch?
+    // Placeholder for instantiating global settings.
+    let investSettings = {};
+    const globalDefaultSettings = getDefaultSettings();
+
+    try {
+      for (const [setting, _val] of Object.entries(globalDefaultSettings)) {
```

Comment:
I like this approach of iterating through the default settings rather than just getting the whole settings store and assuming it has the keys we expect.

Reply:
Thanks!